packages/components/src/components/button/button.css

```diff
@@ -137,8 +139,10 @@
 }
 
 .button.button--icon-after:not(.button--icon-only) ::slotted(*) {
-  margin-left: var(--spacing-icon-x);
-  margin-right: calc(var(--spacing-icon-x) * -1);
+  margin-left: 8px;
```

Comment:
same as above, since spacing-icon-x changes to 8px everywhere we could set: 

`--spacing-icon-x: var(--telekom-spacing-unit-x2);`